e2e/fastify.e2e-spec.ts

```diff
@@ -27,8 +27,8 @@ describe('Fastify Swagger', () => {
       .addBearerAuth()
       .addOAuth2()
       .addApiKey()
-      .addApiKey({ type: 'apiKey' }, 'key1')
-      .addApiKey({ type: 'apiKey' }, 'key2')
```

Comment:
This would make it a breaking change, but would it make more sense to bring the name to be the first parameter instead of the options?

Reply:
@jmcdo29 , yep, you're right. Fixed